lib/registries/addon/components/registries-provider-facet/component.ts

```diff
@@ -38,8 +38,6 @@ export default class RegistriesSourcesFacet extends Component {
 
     @action
     providerChecked(filter: SearchFilter, remove: boolean) {
-        this.analytics.track('filter', remove ? 'remove' : 'add', `Discover - providers ${filter.display}`);
```

Comment:
It looks like the template handles this analytics event now, and this was a duplicate, but I would advocate reinstating this one, and removing the analytics call in the template. we can add some extra params to the analytics call here as well (ie: if it's added or removed, the provider being added/removed, etc)

Reply:
Also would be a good idea to check the other facets (subjects, registration form types) and make sure they are being handled the same way